services/spar/src/Spar/Scim/User.hs

```diff
@@ -324,7 +323,7 @@ createValidScimUser (ValidScimUser user uref handl mbName richInfo) = do
 updateValidScimUser
   :: forall m. (m ~ Scim.ScimHandler Spar)
   => ScimTokenInfo -> UserId -> ValidScimUser -> m (Scim.StoredUser SparTag)
-updateValidScimUser tokinfo@ScimTokenInfo{stiIdP} uid newScimUser = do
+updateValidScimUser tokinfo uid newScimUser = do
```

Comment:
I can't leave comments in places that are not edited by you because Github sucks. Open this file and scroll to line 341:


Does `Scim.getUser` need all of `tokinfo` to work or just a part of `tokinfo`?  Reason I ask is because it's nice if this function only takes what it needs, instead of a `tokinfo` which also contains an `idp` which might lead us to confuse it with the idp in `newScimUser` or `oldScimUser`

Reply:
It's used as an argument for a function from hscim, so I can't think of a way to take the redundant idp-id out of the argument.  You make a good point, though.  If you can think of anything I'd like to hear it.